cpp/data/zotero_cgi/qa.html

```diff
@@ -33,17 +44,17 @@ <h1>QA Settings</h1>
                 </tr>
             </thead>
             <tbody>
-                {LOOP tags, global_settings, journal_settings}
+                {LOOP tags, subfield_codes, regexes, global_settings, journal_settings}
                     <tr>
-                        <td><a href="https://www.loc.gov/marc/bibliographic/bd{tags}.html" title="Link to external MARC 21 documentation at loc.gov" target="_blank">{tags}</a></td>
+                        <td><a href="https://www.loc.gov/marc/bibliographic/bd{tags}.html" title="Link to external MARC 21 documentation at loc.gov" target="_blank">{tags}</a>${subfield_codes}&nbsp;{regexes}</td>
                         <td>{global_settings}</td>
-                        {IF journal_name != ""}<td>{journal_settings}{IF journal_settings != ""}<a href="?action=show_qa&zeder_id={zeder_id}&zeder_instance={zeder_instance}&delete_tag={tags}&delete_type=local" title="Delete this rule" onclick="return confirm('Are you sure?')"><sup>x</sup></a>{ENDIF}</td>{ENDIF}
+                        {IF journal_name != ""}<td>{journal_settings}{IF journal_settings != ""}<a href="?action=show_qa&zeder_id={zeder_id}&zeder_instance={zeder_instance}&delete_tag={tags}&delete_subfield_code={subfield_codes}&delete_type=local" title="Delete this rule" onclick="return confirm('Are you sure?')"><sup>x</sup></a>{ENDIF}</td>{ENDIF}
                     </tr>
                 {ENDLOOP}
             </tbody>
         </table>
 
-        <form id="add_form" method="post" action="?" autocomplete="off" onsubmit="return confirm('Are you sure?')">
+        <form id="add_form" method="post" action="?" autocomplete="off" onclick="ValidateRegex()" onsubmit="return confirm('Are you sure?')">
```

Comment:
If you use the onclick event of the form element, this function will trigger if the user clicks anywhere in the form (any input field, any button, even the background). So you should either move it to the input button or, even better, combine it with the existing onsubmit event.

Reply:
Done